imports/plugins/core/dashboard/client/components/actionView.js

```diff
@@ -189,6 +205,27 @@ class ActionView extends Component {
     }
   }
 
+  componentWillReceiveProps(nextProps) {
+    // Array containing abbreviations of right-to-left languages
+    const rightToLeftLanguages = ["ar", "he"];
```

Comment:
This info should not be hardcoded into a single component, it should be moved to a common file somehow